tests/transports/test_all_plugins.py

```diff
@@ -33,16 +30,29 @@
 # TODO : silly cases of copy/put/get from self to self
 
 
+@pytest.fixture(scope='function')
+def tmp_path_remote(tmp_path_factory):
+    """Mock the remote tmp path using tmp_path_factory to create folder start with prefix 'remote'"""
+    return tmp_path_factory.mktemp('remote')
+
+
+@pytest.fixture(scope='function')
+def tmp_path_local(tmp_path_factory):
+    """Mock the local tmp path using tmp_path_factory to create folder start with prefix 'local'"""
+    return tmp_path_factory.mktemp('local')
+
+
 @pytest.fixture(scope='function', params=entry_point.get_entry_point_names('aiida.transports'))
-def custom_transport(request, tmp_path, monkeypatch) -> Transport:
+def custom_transport(request, tmp_path_factory, monkeypatch) -> Transport:
     """Fixture that parametrizes over all the registered implementations of the ``CommonRelaxWorkChain``."""
     plugin = TransportFactory(request.param)
 
     if request.param == 'core.ssh':
         kwargs = {'machine': 'localhost', 'timeout': 30, 'load_system_host_keys': True, 'key_policy': 'AutoAddPolicy'}
     elif request.param == 'core.ssh_auto':
         kwargs = {'machine': 'localhost'}
-        filepath_config = tmp_path / 'config'
+        # The transport config is store in a indepenednt tmp path to not mix up with the files under operating.
```

Comment:
just to make it more clear:
```suggestion
        # Please note, `tmp_path/transport/config' keeps existing during all the following tests!
```

Reply:
```suggestion
        # The transport config is store in a independent temporary path per test to not mix up with the files under operating.
```

I did a test on printing the config path for every test, it is actually for every test the transport / 'config' is under the folder of its own as `/tmp/pytest-of-jyu/pytest-24/transport7/config` with the number after "transport" incremented. 

I adapt the comment string a bit, hope it is better.